src/main/java/com/iexec/core/replicate/ReplicateModel.java

```diff
@@ -38,7 +38,7 @@ public class ReplicateModel {
     private String resultLink;
     private String chainCallbackData;
     private String contributionHash;
-    private String appStdout;
+    private String appLogs;
 
     public static ReplicateModel fromEntity(Replicate entity) {
         return ReplicateModel.builder()
```

Comment:
`ReplicateModel.builder().computeLogs(...)`

When content is added to `ReplicateModel`, we should purge details to avoid polluting presentation layer.
Something like:
```
for (statusUpate: statusUpdates) {
statusUpdate.setComputeLogs(null);
statusUpdate.setResultLink(null);
...
}
``` 
and add annotation en `ReplicateDetails` to ignore null values

Do you see my point?

Reply:
The best way to do so seems to be adding a new `ReplicateStatusUpdateModel` class. Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/504/commits/815a0ebe6791ab5f9c54a28b72bde30f9b981b75.

Reply:
Excellent, removed an existing `TODO` :clap: 

Do you think
```
    private Integer exitCode; //null means unset
    private String teeSessionGenerationError; // null means unset
```
Can be moved to replicateModel
```
ReplicateStatusUpdateModel.exitCode -> ReplicateModel.appExitCode (like appLogs)
ReplicateStatusUpdateModel.teeSessionGenerationError -> ReplicateModel.teeSessionGenerationError
```
?

Reply:
I have wondered about that. `teeSessionGenerationError` would be easy to be moved to `ReplicateModel` as we are sure there's only one.
Regarding `exitCode`, are we sure we will not want to put pre/post-compute exit codes in our models?

Reply:
Aren't pre & post exit codes useless today since errors are already covered by now business exceptions?

Reply:
I think so, yes. Let's update our code to move `exitCode` and `teeSessionGenerationError` upwards :)

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/504/commits/330a91fc9b1c23feeefcfb3317b68bf3906e0f83.

Reply:
Thanks!